### ISSUE TITLE
Create report file with unknown extensions description and send via email

### ISSUE THREAD

**Issue by hrrmsn** [hrrmsn/ffcleaner#4]
1. At first, use smtp.gmail.com server and 587 port.
2. Use smptd library after that.

**Comment by hrrmsn** [comment]
Maybe it's a good to use mailgun web service.
http://www.mailgun.com/

But i don't know yet can i create attachments for my emails. (Yes, i can. See documentation.)

**Comment by hrrmsn** [comment]
I have a following idea. It's not a good that i want to send a report after each launch of my script. The better solve is send report after each crash of my script. This report will be contain error information and also information about last for example ten launches with unknown files statistics. Data about unknown files statistics can be saved in the special directory which be created by script at the first launch.

**Comment by hrrmsn** [comment]
Report is a log file. Log file should be archived before sending via email. (Zip archive should be create.) After log sending the archive file should be remove.

**Comment by hrrmsn** [comment]
**Done.**

**Comment by hrrmsn** [comment]
I forgot that after sending log must be deleted. (This feature is need to be implemented.)

**Comment by hrrmsn** [comment]
OSError occurs if i delete directory for cleaning when script is still working! ('plungedir' function need to be surrounded by try-except block.)

**Comment by hrrmsn** [comment]
I decided that i need to use a small amount of libraries. (The fewer the better.) So i must send email without using 'requests' library. See following link.
http://stackoverflow.com/questions/33929195/send-an-email-via-mailgun-with-python-httplib-library

**Comment by hrrmsn** [comment]
After some researching i understood that make post request with 'httplib' and 'urllib' usage is not so simple as i thought.

I found in this [article](https://documentation.mailgun.com/api-sending.html#sending) that i can use 'attachment' parameter but then i must use 'multipart/form-data' encoding. And i don't know how to do it. I wrote a message to Garett Chang. Let see what he will have answered. In the worst case i just back to use 'requests' library.
